### PR TITLE
[PBM-948] align describe-backup and describe-restore output

### PR DESCRIPTION
**Pull request by defbin**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Logical | Physical |
| ------------- | ------------- | ------------- |
| New cluster psmdb 4.2 | passed | - |
| Sharded psmdb 4.2 | failed | passed |
| New cluster psmdb 4.4 | passed | - |
| Non-sharded psmdb 4.2 | passed | passed |
| New cluster psmdb 5.0 | passed | - |
| Sharded psmdb 4.4 | failed | failed |
| Sharded psmdb 5.0 | passed | failed |
| Non-sharded psmdb 4.4 | passed | failed |
| Non-sharded psmdb 5.0 | passed | failed |
| Single-node psmdb 4.2 | failed | failed |
| Single-node psmdb 4.4 | failed | failed |
| Single-node psmdb 5.0 | failed | failed |

**Comment by JNKPercona** [comment]
| Test name  | Logical | Physical |
| ------------- | ------------- | ------------- |
| Sharded psmdb 4.2 | passed | failed |
| New cluster psmdb 4.2 | passed | - |
| Sharded psmdb 4.4 | failed | passed |
| New cluster psmdb 4.4 | passed | - |
| New cluster psmdb 5.0 | passed | - |
| Non-sharded psmdb 4.2 | failed | passed |
| Non-sharded psmdb 4.4 | passed | passed |
| Non-sharded psmdb 5.0 | passed | passed |
| Single-node psmdb 4.2 | passed | passed |
| Sharded psmdb 5.0 | passed | passed |
| Single-node psmdb 4.4 | passed | passed |
| Single-node psmdb 5.0 | passed | passed |

File: cli/restore.go [hunk 1]
```diff
@@ -369,37 +369,39 @@ type descrRestoreOpts struct {
...
-	Error              *string       `yaml:"error,omitempty" json:"error,omitempty"`
-	LastTransitionTS   int64         `yaml:"-" json:"last_transition_ts"`
-	LastTransitionTime string        `yaml:"last_transition_time" json:"-"`
-	Nodes              []RestoreNode `yaml:"nodes,omitempty" json:"nodes,omitempty"`
+	Name               string        `json:"name" yaml:"name"`
+	Status             pbm.Status    `json:"status" yaml:"status"`
+	Error              *string       `json:"error,omitempty" yaml:"error,omitempty"`
+	LastWriteTS        int64         `json:"last_write_ts" yaml:"-"`
+	LastTransitionTS   int64         `json:"last_transition_ts" yaml:"-"`
+	LastWriteTime      string        `json:"last_write_time" yaml:"last_write_time"`
```
Comment (dAdAbird) [inline cli/restore.go:394]: Restore doesn't have `LastWriteTime` (no need in L`astWriteTS` for that reason).

**Review by dAdAbird — CHANGES_REQUESTED** [review]

**Review by dAdAbird — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Logical | Physical |
| ------------- | ------------- | ------------- |
| New cluster psmdb 4.4 | failed | - |
| Sharded psmdb 4.2 | failed | failed |
| Non-sharded psmdb 4.2 | failed | failed |
| New cluster psmdb 5.0 | failed | - |
| New cluster psmdb 4.2 | failed | - |
| Sharded psmdb 4.4 | failed | passed |
| Non-sharded psmdb 4.4 | passed | passed |
| Non-sharded psmdb 5.0 | failed | passed |
| Single-node psmdb 4.2 | failed | passed |
| Single-node psmdb 4.4 | failed | passed |
| Sharded psmdb 5.0 | failed | failed |
| Single-node psmdb 5.0 | passed | failed |

**Pull request merged**